rhel/vex/parser_test.go

```diff
@@ -3,14 +3,15 @@ package vex
 import (
 	"bytes"
 	"context"
+	"errors"
 	"io"
 	"net/http"
 	"net/url"
 	"os"
 	"testing"
 
 	"github.com/klauspost/compress/snappy"
-
+	"github.com/package-url/packageurl-go"
 	"github.com/quay/zlog"
```

Comment:
I don't recall other files, but just checking the spacing here

Reply:
I believe this is correct now but was incorrect previously